airflow/providers/daskexecutor/CHANGELOG.rst

```diff
@@ -27,11 +27,29 @@
 Changelog
 ---------
 
-1.1.1
+1.2.0
```

Comment:
See the discussion in https://github.com/apache/airflow-site/pull/907 - I do think we should not bump neither min-airflow version nor MINOR version of the daskexecutor. 

I am open to discuss it, but I think there are good reasons why the daskexecutor  provider should be axactly the same as last released one (with added "removed" description).

Reply:
if https://github.com/apache/airflow/pull/36191 fix the issue then yeah we should stay with 1.1.1